lib/Core/Executor.cpp

```diff
@@ -1102,8 +1104,16 @@ ref<klee::ConstantExpr> Executor::evalConstant(const Constant *c) {
       }
       ref<Expr> res = ConcatExpr::createN(kids.size(), kids.data());
       return cast<ConstantExpr>(res);
+    } else if (const ConstantVector *cv = dyn_cast<ConstantVector>(c)) {
+      ref<Expr> *kids = new ref<Expr>[ cv->getNumOperands() ];
```

Comment:
Don't use C-style new/free allocation, `llvm::SmallVector` is the better choice and gets deallocated if it goes out of scope.
And one line less code ;)